db/schema.rb

```diff
@@ -54,6 +54,7 @@
     t.string   "label",      null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string   "slug"
```

Comment:
I thought about that after I tried to add a not null constraint and it failed because I hadn't rerun the import yet, but thought that could be added and the services migrated at a later date. May as well do it now though now you've mentioned it. 😉